digits/dataset/tasks/create_db.py

```diff
@@ -11,7 +11,7 @@
 from digits.task import Task
 
 # NOTE: Increment this everytime the pickled version changes
-PICKLE_VERSION = 3
+PICKLE_VERSION = 4
```

Comment:
I don't think we need to increment the version here. We can just add this snippet to the end of `__set_state__()`:
```python
if not hasattr(self, 'augmentation'):
    self.augmentation = None
```